prow/config/config.go

```diff
@@ -240,6 +240,8 @@ type Sinker struct {
 	// MaxProwJobAge is how old a ProwJob can be before it is garbage-collected.
 	// Defaults to one week.
 	MaxProwJobAge time.Duration `json:"-"`
+	// MaxPodAge is the maximum duration a pod is kept after finishing
```

Comment:
I don't think this is technically true -- isn't it just the time since it was created? Not after it finishes. Also, this comment already exists on the field below, maybe just copy that if what you're after is Godoc on this field specifically?

Reply:
No (I know that because I looked the code up before): https://github.com/kubernetes/test-infra/blob/73aeb65e21723cdd3bde953981c0a4c6634328e4/prow/cmd/sinker/main.go#L221

But you are right, the docs are right below. Apparently I got too used to "the code is the documentation" and didn't think about looking at the other field. I'll just close this, sorry for the interruption.